recipes/libcugraph/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "libcugraph" %}
+{% set version = "0.19.0a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/rapidsai/cugraph/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: eea8eb788ee2e9f369b6d4911c8cb7c4819453e17aa855eac14af86ee27f627e
+    folder: cugraph
+  # libcugraph depends on the RMM header-only library of the same version.
+  - url: https://github.com/rapidsai/rmm/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: aa2b6542e51e9d3bec1c4053ab2a06af8fb8e9a94be3832418109904ebebb276
+    folder: rmm
+
+build:
+  number: 0
+  skip: true  # [(not linux) or (cuda_compiler_version == "None")]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin="x.x") }}
+  missing_dso_whitelist:
+    - "*/libcuda.*"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+    - cmake >=3.12.4
+    - make
+    - libgomp
+  host:
+    - boost-cpp >=1.66
+    - nccl >=2.8.4
+    - libfaiss =*=*_cuda
+    - spdlog =1.7.0
+    - libgomp
```

Comment:
Remove this

Reply:
Thanks, don't we need `libgomp` here and in `run` since it is linked to and used at runtime?  [Previous error](https://github.com/conda-forge/staged-recipes/pull/14546#discussion_r615154048) and [suggestion](https://github.com/conda-forge/staged-recipes/pull/14546#discussion_r615156263).

Reply:
I removed it from here and `run` below in the next commit, waiting on CI...